### PR TITLE
[4.0] com_menus javascript clean up

### PR DESCRIPTION
**Pull request by dgrammatiko**
Pull Request for Issue # .

### Summary of Changes

- move inline code to files
- Drop some jQuery (modals using jQuery, because why not :( )

### Testing Instructions
- Create a new MENU
- Create all the possible menu items
- Check that modules modals work correctly for Menu and Items

### Documentation Changes Required
None code clean up, preparing joomla for strict CSP mode

### CODE REVIEW

File: media/com_menus/js/admin-item-edit.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
+/**
+ * @copyright  Copyright (C) 2005 - 2016 Open Source Matters, Inc. All rights reserved.
+ * @license    GNU General Public License version 2 or later; see LICENSE.txt
```
Comment (brianteeman) [inline media/com_menus/js/admin-item-edit.js]: 2017 ;)

File: media/com_menus/js/admin-item-edit_container.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+/**
+ * @copyright  Copyright (C) 2005 - 2016 Open Source Matters, Inc. All rights reserved.
```
Comment (brianteeman) [inline media/com_menus/js/admin-item-edit_container.js]: 2017

**Comment by wilsonge** [comment]
Conflicts please

**Comment by dgt41** [comment]
@wilsonge done!

**Pull request merged**